tests/v2/integration/projects/project_user_test.go

```diff
@@ -102,5 +102,6 @@ func (p *ProjectUserTestSuite) TestCreateNamespaceProjectOwner() {
 }
 
 func TestProjectUserTestSuite(t *testing.T) {
+	t.Skip()
```

Comment:
This is a known flaky test. Are we going to skip all the flaky tests? This should be probably be resolved by rerunning it.

Reply:
Just to make sure.